server/portal/apps/workspace/api/views.py

```diff
@@ -298,7 +298,7 @@ def post(self, request, *args, **kwargs):
 
                 # Make sure $HOME/.tap directory exists for user when running interactive apps
                 execSystemId = job_post['execSystemId']
-                system = settings.PORTAL_EXEC_SYSTEMS.get(execSystemId)
+                system = next((v for k, v in settings.TACC_EXEC_SYSTEMS.items() if execSystemId.endswith(k)), None)
```

Comment:
This is actually the only real operational change made. Instead of looking for an exact match in our list of TACC system dir mappings, it looks for a similar string. This will allow us to process system ids `frontera` and `secure.frontera` the same way.